tests/models/test_collection.py

```diff
@@ -885,13 +942,11 @@ def remove_work(self, work):
         # has any LicensePools), but not the second.
         eq_([work], index.removed)
 
-        # The ExternalIntegration and its settings are still around,
-        # since multiple Collections can be based on the same
-        # ExternalIntegration.
```

Comment:
I really don't think this is true. Maybe we thought it might be true when we were writing the Collection/ExternalIntegration code, but we've decided on a system where an Externalintegration is just a convenient way of gathering together some configuration information about a Collection. You could create two Collections with the same credentials (though this sounds like a bad idea) but they would have different underlying ExternalIntegrations.